database/things/awk.pldb

```diff
@@ -3,6 +3,7 @@ appeared 1977
 type pl
 creators Alfred Aho and Peter J. Weinberger and Brian Kernighan
 website http://www.cs.princeton.edu/~bwk/btl.mirror/
+documentation https://www.gnu.org/software/gawk/manual/gawk.html
```

Comment:
This is for the GNU version of AWK. 

There is also the "[One True AWK](https://github.com/onetrueawk/awk)", which more closely follows the _The AWK Programming Language_, by Al Aho, Brian Kernighan, and Peter Weinberger (Addison-Wesley, 1988, ISBN 0-201-07981-X). However, the only documentation is a "Man" page which is not so easily referenced.

Reply:
A useful online version of *awk.1* "man" page is https://www.man7.org/linux/man-pages/man1/awk.1p.html, but it may not correspond exactly with the *[awk.1](https://raw.githubusercontent.com/onetrueawk/awk/master/awk.1)* from the _onetrueawk_ repo.

But it should be a good reference, in any case.

Reply:
Great @celtic-coder , thanks a lot. Also I enabled multiple docs support now so may I add it or replace it?

Reply:
Yes, go ahead and add this as a further documentation reference.

Reply:
Ok for sure @celtic-coder , I will add this too in my next commit. Thanks a lot.